static/js/admin.js

```diff
@@ -667,6 +680,14 @@ function _setup_page() {
                         ui.report_success(i18n.t("Any user may now create new streams!"), create_stream_by_admins_only_status);
                     }
                 }
+                if (response_data.email_changes_disabled !== undefined) {
+                    if (response_data.email_changes_disabled) {
+                        ui.report_success(i18n.t("Users cannot change their email!"), email_changes_disabled_status);
+                    } else {
+                        ui.report_success(i18n.t("Users may now change their email!"), email_changes_disabled_status);
+                    }
+                    toggle_email_change_display();
```

Comment:
Updating the UI should happen inside `server_events.js`; otherwise other browsers won't update properly.